REFERENCE.md

```diff
@@ -44,6 +44,7 @@ These should be set under the `[params]` section:
 | `site_theme`            | No       | The color scheme for the overall site. Currently the options are `orange` (default), `grey`, and `blue`.                                                                                                                                                            | "blue"                                                                                                                                                                                                                                    |
 | `site_layout`           | No       | The layout for the home page. The options are `row` (default) or `grid`.                                                                                                                                                                                            | "grid"                                                                                                                                                                                                                                    |
 | `enable_jumbo`          | No       | When using the `row` layout, will set a jumbotron at the top instead of the sidebar.                                                                                                                                                                                | "true"                                                                                                                                                                                                                                    |
+| `enable_show_next_episode`          | No       | When using the `row` layout, if there are episodes with the upcoming frontmatter set to true, then it will display the next episode on homepage. This halves the screen real estate of the latest episode section, which will then be displayed side by side. | "true"                                                                                                                                                                                                                       |
```

Comment:
I wonder if we could give this a more descriptive name, like `show_next_upcoming` which makes it more clear that it is only for upcoming?

Reply:
Good call, I'll make that change